test/srdf_parser_cpp.test

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <test test-name="test_srdf_parser_cpp" pkg="srdfdom" type="test_cpp">
+    <env name="LC_ALL" value="C" />
+  </test>
+  <test test-name="test_srdf_parser_cpp_locale" pkg="srdfdom" type="test_cpp">
+    <env name="LC_ALL" value="nl_NL.UTF-8" />
```

Comment:
> In order for the first two commits to fail during unit testing somewhere `sudo locale-gen nl_NL.UTF-8` needs to be added, @rhaschke @davetcoleman you probably know where

I don't get this comment. Isn't this env setting correctly setting the locale?

Reply:
yes, but in order for the env variable to have an effect the locale needs to be created first

see https://github.com/ros/urdfdom/commit/1857a55897295c1fa46669bf1483dc78cf2467a8 for an example without using moveit-ci

Reply:
I see. I added an appropriate command to .travis.yml